### PR TITLE
AsteroidOS screenshot

### PR DESCRIPTION
**Pull request by jmlich**
there are few issues so far:
- [x] file contain few extra bytes and screenshot looks corrupted
- [x] files should be stored probably into ~/Pictures
- progress bar is not show in ui (it is not necessary, but poor UX)

### CODE REVIEW

**Comment by jmlich** [comment]
I have spent a lot of time with QDataStream which adds own header.

File: daemon/src/devices/asteroidosdevice.cpp [hunk 1]
```diff
@@ -300,22 +300,19 @@ void AsteroidOSDevice::screenshotReceived(QByteArray data) {
     }
 
     QString fullpath = picturelocation.absolutePath() + "/" + filename;
-    QFile *screenshotFile = new QFile(fullpath);
+    QFile screenshotFile(fullpath);
 
-    qDebug() << fullpath;
+    qDebug() << fullpath << data.size();
 
     QDataStream *dataStream = nullptr;
```
Comment (piggz) [inline daemon/src/devices/asteroidosdevice.cpp:307]: Has this been left over?

Comment (jmlich) [reply daemon/src/devices/asteroidosdevice.cpp:307]: I think file name and size is useful debug message.

**Comment by piggz** [comment]
Is this ready, or are you wanting to add the progress bar part?

**Comment by jmlich** [comment]
I don't want to work on progress bar. It is nice to have feature, but it requires many changes in daemon.

**Comment by piggz** [comment]
Should we have an additional FEATURE flag for this FEATURE_SCREENSHOT, so that the UI can update with whether or not it is available?

**Comment by jmlich** [comment]
flag has been added

**Pull request merged**